transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -216,6 +217,17 @@ public Bootstrap clone() {
         return new Bootstrap(this);
     }
 
+    /**
+     * Returns a deep clone of this bootstrap which has the identical configuration except that it uses
+     * the given {@link EventLoopGroup}. This method is useful when making multiple {@link Channel}s with similar
+     * settings.
+     */
+    public Bootstrap clone(EventLoopGroup group) {
```

Comment:
I was thinking about this but not sure we want to support this for ServerBootstrap as well. Let me think a bit more about it.